### PR TITLE
feature: new responses for confession command

### PR DESCRIPTION
**Pull request by dragonyc1002**
## Changes
* Add more responses for confession command([Confession.ts](./src/commands/Confession.ts)) (d0fb5a4).

### CODE REVIEW

File: src/commands/Confession.ts [hunk 1]
```diff
@@ -152,14 +152,18 @@ export default class Confession extends Command<[GuildMember]> {
       '要我接受你請等一萬年後再說', '你自我感覺會不會太好', '滾', '找點有意義的事做行不行',
       '拒絕', '收回你的妄想', '慢走不送', '我建議你先照個鏡子', '你是個好人', '駁回',
       '請讓我再想想', '我會考慮的', '你為什麼要做這種吃力不討好的事', '我建議你去睡覺，夢裡什麼都有',
-      '門都沒有', '我接受你是可能事件，但機率為 0', '謝謝', '出口在左手邊', '你會不會太高估你自己'
+      '門都沒有', '我接受你是可能事件，但機率為 0', '謝謝', '出口在左手邊', '你會不會太高估你自己', '你也太慘了吧，淪落到要來找機器人告白，但是你根本就不夠格，掰:D',
```
Comment (kyometori) [inline src/commands/Confession.ts:155]:
```suggestion
      '門都沒有', '我接受你是可能事件，但機率為 0', '謝謝', '出口在左手邊', '你會不會太高估你自己', '你也太慘了吧，淪落到要來找機器人告白，但是你根本就不夠格，掰 :D',
```

File: src/commands/Confession.ts [hunk 2]
```diff
@@ -152,14 +152,18 @@ export default class Confession extends Command<[GuildMember]> {
...
+      '當你罵我爛bot的時候，你有考慮過我的感受嗎？沒有？那你還來跟我告白什麼？當我是你的工具？你以為自己多厲害？天理難容，出口在右手邊',
```
Comment (kyometori) [inline src/commands/Confession.ts:156]:
```suggestion
      '當你罵我爛 bot 的時候，你有考慮過我的感受嗎？沒有？那你還來跟我告白什麼？當我是你的工具？你以為自己多厲害？天理難容，出口在右手邊',
```

**Review by kyometori — CHANGES_REQUESTED** [review]
半形英數符號和中文非標點間應加上一個半形空格

**Comment by dragonyc1002** [comment]
已經將半形空格補上

**Review by Zollo757347 — APPROVED** [review]

**Review by kyometori — APPROVED** [review]

**Pull request merged**